circuits/ts/__tests__/utils/utils.ts

```diff
@@ -1,4 +1,16 @@
-import type { WasmTester } from "circom_tester";
+import { Circomkit, type WitnessTester, type CircomkitConfig } from "circomkit";
+
+import fs from "fs";
+import path from "path";
+
+const configFilePath = path.resolve(__dirname, "..", "..", "..", "circomkit.json");
+const config = JSON.parse(fs.readFileSync(configFilePath, "utf-8")) as CircomkitConfig;
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
Why do you need this rule? It should be disabled in linter.

Reply:
oh yea that was copy paste from another repo, will remove